src/main/java/lery/Ui.java

```diff
@@ -0,0 +1,46 @@
+package lery;
+
+/**
+ * Represents a Ui.
+ * The Ui class handles the user interface components and messages for the Lery chatbot application.
+ * It provides methods to print formatted messages and the Lery logo.
+ *
+ * The class includes methods for printing messages with horizontal lines, greeting messages,
+ * and exit messages.
+ *
+ */
+public class Ui {
+    private static final String LINE = "____________________________________________________________\n";
```

Comment:
```suggestion
    private static final String BORDER = "_".repeat(50);
```

I think it would help to make this into a constant with fixed width like so